packages/synapse-react-client/src/lib/umd.index.ts

```diff
@@ -66,7 +67,8 @@ import { FullContextProvider } from './utils/FullContextProvider'
 import './style/main.scss'
 
 const SynapseContext = {
-  SynapseContextProvider: FullContextProvider,
+  FullContextProvider,
+  SynapseContextProvider,
```

Comment:
I changed my mind on using an alias here too. Will have to make the change in SWC.